core/src/test/java/org/pentaho/metaverse/impl/VfsLineageCollectorTest.java

```diff
@@ -0,0 +1,139 @@
+/*! ******************************************************************************
+ *
+ * Pentaho Data Integration
+ *
+ * Copyright (C) 2002-2017 by Pentaho : http://www.pentaho.com
+ *
+ *******************************************************************************
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ******************************************************************************/
+
+package org.pentaho.metaverse.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.anyInt;
+import static org.mockito.Mockito.atLeast;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.util.List;
+
+import org.apache.commons.io.FilenameUtils;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * Created by rfellows on 7/6/15.
+ */
+public class VfsLineageCollectorTest {
+
+  VfsLineageCollector collector;
+
+  @Before
+  public void setUp() throws Exception {
+    collector = new VfsLineageCollector();
+    String basePath = new File( "." ).getCanonicalPath();
+    collector.setOutputFolder( FilenameUtils.separatorsToSystem( "file://" + basePath
+        + "/src/test/resources/pentaho-lineage-output" ) );
```

Comment:
should never write to a source controlled folder for tests.
write to target/<something> instead. then it will get ignored by git and cleaned up when you run`mvn clean`